pkg/operators/localmanager/localmanager.go

```diff
@@ -293,6 +301,11 @@ func (l *localManagerTrace) PreGadgetRun() error {
 	id := uuid.New()
 	l.subscriptionKey = id.String()
 	host := l.params.Get(Host).AsBool()
+	systemd := l.params.Get(Systemd).AsBool()
+
+	if !host && systemd {
+		return fmt.Errorf("flag --%s can only be used with --%s", Systemd, Host)
+	}
```

Comment:
I also thought about making both implicit - But I wanted to present it here for discussion. Also like the implication more

Reply:
`--show-systemd` now implies `--host`.
But I can't make it so that `--sdunit sshd.service` implies `--show-systemd` because of the automatic column filtering I do here. It is required that the param is set to true